src/clojure_tictactoe/game/game_play.clj

```diff
@@ -8,27 +8,28 @@
             [clojure-tictactoe.game.game-rules :as rules]
             ))
 
-(defn player-move [move-function board]
-  (board/mark-space (move-function board) @players/player-atom board))
+(defn player-move [move-function board marker]
+  (board/mark-space (move-function board) marker board))
+
+(defn continue-game [board markers turn]
+  (let [new-board (player-move (players/choose-player-function turn) board (markers (players/current-player turn)))]
 
-(defn continue-game [board]
-  (let [new-board (player-move (players/choose-player-function @players/player-atom) board)]
     (board-printer/print-board new-board)
-    (players/player-swap)
     new-board))
 
-(defn game-loop [board]
-  (loop [board board]
-    (if-not (rules/game-over? board)
-      (let [new-board (continue-game board)]
-        (recur new-board))
-      (if-let [results (rules/assess-winner board)]
-        (println (end-printer/game-won-message results))
-        (println end-printer/game-tie-message)))))
+(defn game-loop [board markers turn]
+  (if-not (rules/game-over? board)
+    (let [new-board (continue-game board markers turn)]
+
+      (recur new-board markers (+ 1 turn)))
+    (if-let [results (rules/assess-winner board)]
+      (println (end-printer/game-won-message results))
+      (println end-printer/game-tie-message))))
```

Comment:
Re: the use of `println`, should `game-play` know that the UI is a console?

Reply:
Cool, I'll work this in with the next stories.